.github/workflows/advanced.yml

```diff
@@ -66,12 +66,18 @@ jobs:
 
     strategy:
       matrix:
-        target: [aarch64-w64-mingw32, x86_64-w64-mingw32]
+        arch: [aarch64, x86_64]
+        platform: [w64-mingw32, linux-gnu]
         crt: [msvcrt, ucrt]
```

Comment:
Please add another value to the `crt` list, `libc`, and exlude all combinations `w64-mingw32` and `libc` from the matrix.

Reply:
Done